website_demo/index.html

```diff
@@ -0,0 +1,326 @@
+<!doctype html>
+<html class="no-js" lang="en">
+
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <title>How are you really</title>
+        <meta name="description" content="We would like to invite you to our demo day.">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <meta name="theme-color" content="#ffffff">
+        <meta property="og:title" content="How are you really">
+        <meta property="og:description" content="We would like to invite you to our demo day.">
+        <meta property="og:videoURL" content="https://www.youtube.com/watch?v=3VLJO-izWPA">
+        <meta property="og:type" content="website">
+        <link rel="apple-touch-icon" sizes="180x180" href="apple-touch-icon.png">
+        <link rel="icon" type="image/png" sizes="32x32" href="hcp_logo_32.png">
+        <link rel="icon" type="image/png" sizes="16x16" href="hcp_logo_16.png">
+        <link rel="manifest" href="manifest.json">
+        <link rel="mask-icon" href="safari-pinned-tab.svg" color="#5bbad5">
+        <link rel="stylesheet" href="css/normalize.min.css">
+        <link rel="stylesheet" href="css/bootstrap.min.css">
+        <link rel="stylesheet" href="css/jquery.fancybox.css">
+        <link rel="stylesheet" href="css/flexslider.css">
+        <link rel="stylesheet" href="css/styles.min.css">
+        <link rel="stylesheet" href="css/queries.css">
+        <link rel="stylesheet" href="node_modules/animate.css/animate.min.css">
+        <link rel="stylesheet" href="node_modules/font-awesome/css/font-awesome.min.css">
+        <script src="js/vendor/modernizr-2.8.3-respond-1.4.2.min.js"></script>
+    </head>
+
+    <body id="top">
+
+        <section class="hero">
+            <section class="navigation">
+                <header>
+                    <div class="header-content">
+                        <div class="logo"><a href="#"><img src="img/hcp_logo_64.png" alt="Logo"></a></div>
+                        <div class="header-nav">
+                            <nav>
+                                <ul class="primary-nav">
+                                    <li><a href="#intro">About Us</a></li>
+                                    <li><a href="#events">Demo</a></li>
+                                    <li><a href="#instagram"></a></li>
+                                </ul>
+                                <ul class="member-actions">
+                                    <!-- <li><a href="#map" class="login">Venue</a></li> -->
```

Comment:
There are some commented out code that you forgot to remove! Please check the rest of the file as well.